prod/bsc/feeds-5.5.0.yaml

```diff
@@ -0,0 +1,9938 @@
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WMT
+
+EQ:WRB:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WRB
+
+EQ:WRK:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WRK
+
+EQ:WST:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WST
+
+EQ:WU:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WU
+
+EQ:WY:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WY
+
+EQ:WYNN:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: WYNN
+
+EQ:XEL:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: XEL
+
+EQ:XLNX:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: XLNX
+
+EQ:XOM:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: XOM
+
+EQ:XRAY:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: XRAY
+
+EQ:XRX:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: XRX
+
+EQ:XYL:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: XYL
+
+EQ:YUM:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: YUM
+
+EQ:ZBH:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: ZBH
+
+EQ:ZBRA:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: ZBRA
+
+EQ:ZION:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: ZION
+
+EQ:ZTS:
+  discrepancy: 0.5
+  precision: 2
+  inputs:
+    - fetcher:
+        name: PolygonIOPrice
+        params:
+          sym: ZTS
+
+CPI-US-3months:
+  discrepancy: 1.0
+  precision: 1
+  inputs:
+    - fetcher:
+        name: BEACPIAverage
+        params:
+          months: 3
+
+# yearn.finance
+
+YEARN-TEST:*-ETH:
+  discrepancy: 1
+  precision: 6
+  inputs:
+    - fetcher:
+        name: YearnVaultTokenPrice
+        params:
+          address: "0x0E928957572875954687BcA992fBA397642d0511"
+          network: eth-ropsten
+      calculator:
+        name: YearnTransformPrice
+        params:
+          tsym: ETH
```

Comment:
Will we need this for prod? Please confirm with AV. Does this post ropsten stuff on mainnet?